tests/endtoend/eventhub_batch_functions/get_eventhub_batch_triggered/function.json

```diff
@@ -1,27 +1,27 @@
-{
-  "scriptFile": "__init__.py",
-  "bindings": [
-    {
-      "type": "httpTrigger",
-      "direction": "in",
-      "authLevel": "anonymous",
-      "methods": [
-        "get"
-      ],
-      "name": "req"
-    },
-    {
-      "direction": "in",
-      "type": "table",
-      "name": "testEntities",
-      "partitionKey": "WillBePopulated",
-      "tableName": "EventHubBatchTest",
-      "connection": "AzureWebJobsStorage"
-    },
-    {
-      "type": "http",
-      "direction": "out",
-      "name": "$return"
-    }
-  ]
```

Comment:
Sorry - is the change only in the tab spaces?

Reply:
Yes, this change is tab spaces only.